apps/web/src/components/templates/chat-editor/TemplateChatEditor.tsx

```diff
@@ -24,6 +25,7 @@ export function TemplateChatEditor({
   return (
     <>
       {!isIntegrationActive ? <LackIntegrationError channelType={ChannelTypeEnum.CHAT} /> : null}
+      <VariableManager template={`steps.${index}.template`} />
```

Comment:
A small question, is the variables shared across all the different templates?

Reply:
No, it's bound to each step.

Reply:
Awesome this makes a lot of sense. Maybe later we could also add a global view of the combined variables from all inputs on the workflow editor. But for now it's totally perfect.